ACPs/83-dynamic-multidimensional-fees/README.md

```diff
@@ -47,24 +48,40 @@ $$base \  fee = \sum_{i=0}^3 r_i \times u_i$$
-We seek a scheme that rapidly increases the fees when blocks complexity goes above a defined threshold and that equally rapidly decreases the fees once complexity goes down (because blocks carry less/simpler transactions, or because they are produced more slowly).
 
-Suppose that a block $B$ is the current chain tip. $B$ has the following features:
+We seek a scheme that rapidly increases the fees when blocks complexity goes above a defined threshold and that equally rapidly decreases the fees once complexity goes down (because blocks carry less/simpler transactions, or because they are produced more slowly). We define the desired threshold as a *target complexity rate* $T_i$: we would want to process every second a block whose complexity is $T_i$. Any complexity more than that causes some congestion that we want to penalize via fees; any congestion below $T_i$ causes the network to be under utilized.
+
+In order to update fees rates we will track, per each block and each fee dimension, a parameter called cumulative excess complexity. Fee rates applied to a block will be defined in terms of cumulative excess complexity as we show in the following.
+
+Let's define cumulative excess complexity first.
+
+Suppose that a block $B_t$ is the current chain tip. $B_t$ has the following features:
 
 - $t$ is its timestamp.
-- $B_i$ is the cumulated complexity of all its transactions along the fee dimension $i$.
-- $r_{i,t}$ is the fee rate applied to all of its transactions along the fee dimension $i$.
+- $Excess_{i,t}$ is the cumulative excess complexity along fee dimension $i$.
+
+Say a new block $B_{t + \Delta T}$ is accepted on top of $B$, with the following features:
+
+- $t + \Delta T$ is its timestamp
+- $B_i$ is its complexity.
+
+Then cumulative excess complexity once $B_{t + \Delta T}$ is accepted is updated as follow:
 
-A new block is incoming, whose timestamp is $t + \Delta T$.  
-We wish to maintain a *target complexity rate* $T_i$, i.e. we would want to process every second a block whose cumulated complexity is $T_i$. Any complexity more than that causes some congestion that we want to penalize via fees; any congestion below $T_i$ causes the network to be under utilized.
+$$Excess_{i,t + \Delta T} = max\large(0, Excess_{i,t} - T_i \times \Delta T\large) + B_i$$
 
-The fee rates $r_{i,t+\Delta T}$ applied to the incoming blocks are defined with the following *update formula*:
+Finally let's define the fee rates update formula.
 
-$$r_{i,t+\Delta T} = max \bigl( r^{min}_i, e^{k_i \frac{B_i-T_i \times \Delta T}{T_i \times \Delta T}} \bigr)$$
+Say block $B_t$ is chain tip and a block $B_{t + \Delta T}$ is incoming on top of it. Then
 
-where $r^{min}_i$ and $k_i$ are the minimal fee rate allowed and the update constants along dimension $i$ respectivelly.
+$$ r_{i,t + \Delta T} = r^{min}_i \times e^{\frac{Excess_{i,t} - T_i \times \Delta T}{Denom_i}} $$
```

Comment:
this equation isn't rendered correctly in the preview, not 100% sure the suggestion will change that  
```suggestion
$$r_{i,t + \Delta T} = r^{min}_i \times e^{\frac{Excess_{i,t} - T_i \times \Delta T}{Denom_i}}$$
```

Reply:
Thanks for spotting this. I have simplified the formula (dropping some subscripts) to fix the rendering and it should be good now.